### ISSUE TITLE
Activity level repository.

### ISSUE THREAD

**Issue by shakeel30** [shakeel30/OpenStreetMap-Class-Activity#1]
@rajsukanya hi.
I based the activity levels on the "iD editor" project repository. 
I'm gonna look to see if there's a source code in the repo we can use.

**Comment by shakeel30** [comment]
I found a code base in the iD repo that I think we can use as the source code.
It's is Java script though. Below is the link.

https://github.com/openstreetmap/iD/tree/master/modules/core

There's about seven files in there, each with code.
I'm gonna go ahead and count the number of lines in each file, and enter the sum in the project evaluation.

**Comment by rajsukanya** [comment]
@shakeel30 That's great! Thanks
I'm sorry you had to do the major of the work. I wasn't able to work on this until today. Thank you

**Comment by rajsukanya** [comment]
@shakeel30 In the file context.js, (https://github.com/openstreetmap/iD/blob/master/modules/core/context.js) there are comments (starts at line 311) like: /* Graph */, /* Modes */, /* Behaviors */, /* Map */, etc. (This is code and has those comments above a block of code). Can this file be considered to contain the documentation about the design?

Also I just want to make sure, I see that several issues have tags "bug". Would it be considered triaged?

Sorry, one more question: for the question regarding how many people are maintaining the project - I checked the closed pull requests and saw about two people (quincylvania and bhousel) merging the commits, are they the maintainers? or I also found 20 people under OpenStreetMap, https://github.com/orgs/openstreetmap/people?utf8=%E2%9C%93&query=

**Comment by shakeel30** [comment]
Oh it's no problem at all. Happy to help.

**Comment by shakeel30** [comment]
In reference to the first link, I don't think those comments passes as documentation on *Design* so lets just put no.

As for whether they are triaged, I'm not entirely sure what that means. It's why I left that question blank under Activity level. I'm confused because he asks in the previous question if they are tagged. Not sure what the differences are.

**Comment by shakeel30** [comment]
It lists those 20 people as just members of OpenStreetMap. 
I think (quincylvania and bhousel) are closer to being maintainers since they are authorized to merge pull request.
Thanks for the info.

**Comment by rajsukanya** [comment]
Ok thanks!

**Comment by rajsukanya** [comment]
For the question asking if they are triaged, I checked other project evaluations and these are some of the responses they wrote:
"Yes, there are issues labeled as "major" and "bugs" indicating that they must be worked on."
"yes, their issues have tags and they are listed by priority"
"n order to classify the significance of a particular issue, a labeling system similar to that implemented for denoting difficulty is used in the form of "impact" e.g. impact: low, impact:medium, and impact:high."
"No, none are tagged as triaged."
"Yes, they are on codetriage.com."

**Comment by rajsukanya** [comment]
I checked codetriage.com for openstreetmap https://www.codetriage.com/openstreetmap/openstreetmap-website
But it's in ruby and for the website.. This is what is says: 
The easiest way to get started contributing to Open Source ruby projects like openstreetmap-website
So should we say yes or no?

**Comment by shakeel30** [comment]
> For the question asking if they are triaged, I checked other project evaluations and these are some of the responses they wrote:
> "Yes, there are issues labeled as "major" and "bugs" indicating that they must be worked on."
> "yes, their issues have tags and they are listed by priority"
> "n order to classify the significance of a particular issue, a labeling system similar to that implemented for denoting difficulty is used in the form of "impact" e.g. impact: low, impact:medium, and impact:high."
> "No, none are tagged as triaged."
> "Yes, they are on codetriage.com."

That's great!
Okay, then from what I can tell, the issues are indeed triaged. And they explained it all in the CONTRIBUTION.MD file.

https://github.com/openstreetmap/iD/blob/master/CONTRIBUTING.md

**Comment by shakeel30** [comment]
> I checked codetriage.com for openstreetmap https://www.codetriage.com/openstreetmap/openstreetmap-website
> But it's in ruby and for the website.. This is what is says:
> The easiest way to get started contributing to Open Source ruby projects like openstreetmap-website
> So should we say yes or no?

Lets say yes.

**Comment by shakeel30** [comment]
Hi Raj
I Went ahead and added a summary at the end of the file.
Please feel free to make any changes or additions you deem fit. 
Thanks.

**Comment by rajsukanya** [comment]
I completely forgot, thank you for adding the summary

**Comment by rajsukanya** [comment]
I have a question regarding our own project evaluations. Where do we find a project to evaluate? Is it from the list he provided for the in-class team project evaluations we did?

**Comment by shakeel30** [comment]
> I have a question regarding our own project evaluations. Where do we find a project to evaluate? Is it from the list he provided for the in-class team project evaluations we did?

No, we are not to use any of the projects in that list. You can just use any random open source project that has a repository on github. Also it has to be one that no one else has already picked.

You can find some of them here: https://www.quora.com/What-are-some-simple-C++-open-source-projects-on-GitHub-for-a-beginner

Or you can just google "open source projects on github" and evaluate your pick.

**Comment by rajsukanya** [comment]
> > I have a question regarding our own project evaluations. Where do we find a project to evaluate? Is it from the list he provided for the in-class team project evaluations we did?
> 
> No, we are not to use any of the projects in that list. You can just use any random open source project that has a repository on github. Also it has to be one that no one else has already picked.
> 
> You can find some of them here: https://www.quora.com/What-are-some-simple-C++-open-source-projects-on-GitHub-for-a-beginner
> 
> Or you can just google "open source projects on github" and evaluate your pick.

Thank you so much!

**Comment by shakeel30** [comment]
> I have a question regarding our own project evaluations. Where do we find a project to evaluate? Is it from the list he provided for the in-class team project evaluations we did?
> 
> No, we are not to use any of the projects in that list. You can just use any random open source project that has a repository on github. Also it has to be one that no one else has already picked.
> You can find some of them here: https://www.quora.com/What-are-some-simple-C++-open-source-projects-on-GitHub-for-a-beginner
> Or you can just google "open source projects on github" and evaluate your pick.
> 
> Thank you so much!

You're welcome.
